### ISSUE TITLE
Is this a clerical error in readme.md? 

### ISSUE THREAD

**Issue by ImSiegeLion** [Snipa22/nodejs-pool#399]
I followed the guid in `readme.md` and I encountered a question that puzzled me.
In line 116 of the `readme.md` file reads:
`pool/feeAddress`
But in line 215 of the `deployment/base.sql` file reads:
`INSERT INTO pool.config (module, item, item_value, item_type, Item_desc) VALUES ('payout', 'feeAddress', '', 'string', 'Address that pool fees are sent to.');`

Is this a clerical error in readme.md?  or is my misunderstanding?

**Comment by ImSiegeLion** [comment]
That's right. But in `/sample_config.sql`, the `module` is also `payout`, i think it should be based on the file, right?

**Comment by BKdilse** [comment]
I see what you mean.
I believe it's a typo in Readme:

> pool/feeAddress

 should read payout/feeAddress
